src/main/resources/application.properties

```diff
@@ -0,0 +1,11 @@
+###### Email Properties ######
+spring.mail.host = smtp.gmail.com
+spring.mail.port = 587
+spring.mail.properties.mail.smtp.starttls.enable = true
+spring.mail.username = covidtracking08@gmail.com 
```

Comment:
Danger! I don't think these properties should be stored in a file that gets version controlled. And especially not the password!!! Someone will hack your account! At least as far as the email address is concerned (and maybe other properties as well), these should be stored as systems properties on your machine, and similarly as system properties on the deploy side. Heroku provides a place where you can set whatever properties you need.